CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2021-08-10
```

Comment:
When you change the changelog, you don't need to add a date yet, just add the new entries on top. (there should actually be a title `Next release` to group them,

When we do a release, we add the title with the current date.